python/src/aiconfig/eval/common.py

```diff
@@ -1,14 +1,21 @@
 import json
 from abc import ABC, abstractmethod
 from dataclasses import dataclass
-from typing import Any, Generic, Protocol, TypeVar
+from typing import Any, Generic, NewType, Protocol, Type, TypeVar
 
 import lastmile_utils.lib.core.api as cu
-from pydantic import root_validator
+import result
+from aiconfig.eval import common
```

Comment:
Isn't this import referring to this module/file?

Reply:
Yes, in this case I'm using sort of partly-qualified identifiers like `common.SerializedJSON`. This is often better to disambiguate, e.g. it's the standard way to use pandas/numpy/pytorch/etc. Unfortunately vscode's auto-import fix does it the other way, so this is done inconsistently.